bin/init-cluster

```diff
@@ -74,7 +74,7 @@ echo "K3d cluster configuration exported to ${KUBECONFIG}"
 # Pre-provision PersistentVolumes with RWX accessModes
```

Comment:
In your commit message: `s/PV pre-previsionning/PV pre-provisionning/`

Reply:
Thanks!